task/backend/bolt/bolt.go

```diff
@@ -87,7 +95,11 @@ func New(db *bolt.DB, rootBucket string) (*Store, error) {
 	if err != nil {
 		return nil, err
 	}
-	return &Store{db: db, bucket: bucket, idGen: snowflake.NewDefaultIDGenerator()}, nil
+	st := &Store{db: db, bucket: bucket, idGen: snowflake.NewDefaultIDGenerator()}
```

Comment:
Seems like `latestReplace` should be initialized to `math.MinInt64`. Otherwise it's plausible that a task that was created to run over historical data will "fast-forward" to zero time.